src/client/components/MegaMenu/MegaMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import { Heading, LinkList, MegaMenu } from '@amsterdam/design-system-react';
+import { ChapterMenuItem } from '../../../universal/config';
+import styles from './MegaMenu.module.scss';
+import { MenuItem } from '../MainHeader/MainNavBar.constants';
+
+type Props = {
+  menuItems: MenuItem[];
+  chapters: ChapterMenuItem[];
+};
+
+export default function Menu({ menuItems, chapters }: Props) {
+  return (
+    <MegaMenu>
+      <div className={styles.menu}>
+        <Heading level={3} size="level-3">
+          Thema’s
+        </Heading>
+
+        <Heading level={3} size="level-3">
+          Categorieën
+        </Heading>
+
+        <MegaMenu.ListCategory>
+          <LinkList>
+            {chapters.map((chapter) => (
+              <LinkList.Link key={chapter.id} href={chapter.to}>
```

Comment:
Dit werkt niet in SPA. OnClick handler of MARouterlink gebruiken voor SPA navigatie.